site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -64,6 +65,58 @@ spec:
     annotations:
       map:  garden22
   {{< /tab >}}
+{{% /feature %}}
+{{% feature publishVersion="1.34.0" %}}
+{{< tabpane >}}
+  {{< tab header="selectors" lang="yaml" >}}
+apiVersion: "allocation.agones.dev/v1"
+kind: GameServerAllocation
+spec:
+  # GameServer selector from which to choose GameServers from.
+  # Defaults to all GameServers.
+  # matchLabels, matchExpressions, gameServerState and player filters can be used for filtering.
+  # See: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for more details on label selectors.
+  # An ordered list of GameServer label selectors.
+  # If the first selector is not matched, the selection attempts the second selector, and so on.
+  # This is useful for things like smoke testing of new game servers.
+  selectors:
+    - matchLabels:
+        agones.dev/fleet: green-fleet
+        # [Stage:Alpha]
+        # [FeatureFlag:PlayerAllocationFilter]
+      players:
+        minAvailable: 0
+        maxAvailable: 99
+    - matchLabels:
+        agones.dev/fleet: blue-fleet
+    - matchLabels:
+        game: my-game
+      matchExpressions:
+        - {key: tier, operator: In, values: [cache]}
```

Comment:
I might be not seeing something, but shouldn't `gameServerState` be in here somewhere?

Reply:
Added it along with description